Bedford-PD/bedford_complaints.yaml

```diff
@@ -0,0 +1,543 @@
+
+entityDefinitions:
+
+  person:
+    fqn: "general.person"
+    entitySetName: "MiddlesexBedfordPeople"
+    propertyDefinitions:
+      nc.SubjectIdentification:
+        type: "nc.SubjectIdentification"
+        column: "DEFENDANT_MSTR_NO"
+    conditions:
+    - !<conditions.BooleanIsNullCondition>
+      column: "DEFENDANT_MSTR_NO"
+      reverse: True
+    name: "person"
+
+  offenseevent:
+    fqn: "ol.chargeevent"
+    entitySetName: "MiddlesexBedfordOffenseEvents"
```

Comment:
Are we using offenseevent and chargeevent interchangeably?

Reply:
(I mean in the EDM, I understand their conceptual difference)

Reply:
The plan is to use chargeevent for both and make the meaning clear in the entity set title/other metadata and (if necessary) in a property as well. "Offense event" is more general than "charge event", but we're going with `ol.chargeevent` for backward compatibility